lib/agent/cloud/common.go

```diff
@@ -140,3 +141,16 @@ func StoreCloudConfig(configReponse []byte) bool {
 	ApplyCloudConfig()
 	return true
 }
+
+func LogCloudRequestError(text string, err error) {
+	if atomic.LoadUint32(&globals.GotTraffic) == 0 {
+		return
+	}
+	if err.Error() == "no token set" {
```

Comment:
let's comment what the goal is. Feels like it swallows the error under certain conditions?